aioftp/client.py

```diff
@@ -120,7 +120,8 @@ def __init__(self,
                  read_speed_limit=None,
                  write_speed_limit=None,
                  path_timeout=None,
-                 path_io_factory=pathio.PathIO):
+                 path_io_factory=pathio.PathIO,
+                 list_compatibility_level=2):
```

Comment:
I think we don't need "compatibility level", since user should know nothing about `MLSD` or `LIST`, user want to call `client.list()` and that is all. So, implementation should be based on «unknown command» response and fallback to `LIST`.